coverage/src/BamViewChart.js

```diff
@@ -106,17 +92,26 @@ function createBamView(bamHeader, data, element, broker) {
                 .attr('class', 'circle-button-reset-chromosomes chromosome-button')
                 .attr('transform', 'translate(30, 20)')
                 .on('click', (event, d) => {
-                    // Clear the input fields
-                    bamViewControlsElement.querySelector('#bamview-region-chromosome').value = '';
-                    bamViewControlsElement.querySelector('#bamview-region-start').value = '';
-                    bamViewControlsElement.querySelector('#bamview-region-end').value = '';
-                    bamViewControlsElement.querySelector('#gene-name-input').value = '';
-                // Redraw the chart
-                drawChart(svg);
-
-                // Dispatch custom event from the shadow DOM element
-                dispatchCustomEvent('selected-regions-change', bamHeader);
-            });
+                    // Redraw the chart
+                    drawChart(svg);
+
+                    // Dispatch custom event from the shadow DOM element
+                    dispatchCustomEvent('selected-regions-change', bamHeader);
+
+                    // Dispatch custom event for resetting the region inputs
+                    const regionsInput = {
```

Comment:
What I was trying to say earlier is that we want this event to use the same data type as `selected-regions-change`. It's defined here: https://github.com/iobio/iobio-charts/blob/d5aabc423a138443b1efc3294b1cb0c403ada9ee/data_broker.js#L5

Reply:
@anderspitman Sorry, I am not clear what you mean by that. It looks like you set up the regions as an object type. Here the regionsInput is also an object, right? Or I am understanding in a wong way.

Reply:
You want it to look like this:
```javascript
{
  rname: "<rname>",
  start: <start int>,
  end: <end int>,
}
```

Reply:
Do you mean converting the rname to string type, the start and end to integer type here?

Reply:
I don't think you need to convert anything. This is the native format for all those values.